addons/bestja_project/views/project.xml

```diff
@@ -7,8 +7,10 @@
             <field name="view_mode">kanban,tree,form</field>
             <field name="domain">[
                 '|',
-                    ('organization.coordinator.id', '=', uid),
-                    ('manager.id', '=', uid),
+                    ('organization.coordinator', '=', uid),
+                '|',
```

Comment:
Does this `|` change anything, since there is another `|` at the beginning of the domain?